packages/tools/devtools/devtools-core/src/Features.ts

```diff
@@ -51,4 +51,20 @@ export interface ContainerDevtoolsFeatureFlags {
 	 * Indicates that the Container Devtools supports visualizing the data associated with the Container.
 	 */
 	containerDataVisualization?: boolean;
+
+	/**
+	 * Indicates that the Container Devtools supports modifying the container's connection and lifecycle state.
+	 *
+	 * @remarks
+	 * When `true`, the devtools instance supports:
+	 * - **Connect**: Connecting a disconnected container
+	 * - **Disconnect**: Disconnecting a connected container
+	 * - **Close**: Permanently closing/disposing the container
+	 *
+	 * When `false`, these operations are not available (typical for data object devtools
+	 * which don't have direct control over container lifecycle).
```

Comment:
The note here still referenced "data object devtools", but I also don't think it's necessary. The first part clearly spells out the contract.
```suggestion
	 * When `false`, these operations are not available.
```